lib/puma/binder.rb

```diff
@@ -155,6 +156,14 @@ def parse(binds, logger, log_msg = 'Listening')
 
           @listeners << [str, io]
         when "ssl"
+
+          unless @@minissl_loaded
+            require 'puma/minissl/context_builder'
+            require 'puma/accept_nonblock'
+            require 'puma/minissl'
```

Comment:
These two require lists differ, but use the same guard variable. Should they be equivalent? Is it worth guarding — is requiring an already-required path an expensive enough operation balanced against how often these paths are called?

Reply:
Thank you for reviewing and my apologies.  That is wrong; it was a last minute change for JRuby.

As to the guard, I guess Puma isn't binding all the time, but require will always do a directory search so it loads the first matching file in $LOAD_PATH.

Fixed.